packages/create-block/lib/templates.js

```diff
@@ -161,9 +161,11 @@ const getPluginTemplate = async ( templateName ) => {
 
 	try {
 		if ( existsSync( resolve( templateName ) ) ) {
-			return await configToTemplate( require( resolve( templateName ) ) );
+			return await configToTemplate(
+				await require( resolve( templateName ) )
```

Comment:
As I mentioned to you in chat, I don't see how this can work because `require` is synchronous? I could be missing something.

Reply:
The returned values is a self invoked function.